explainers/content-addressable-bundles.md

```diff
@@ -0,0 +1,429 @@
+# Explainer: Resource loading with Content Addressable Bundles
+
+Last update: Mar 2021
+
+This is a strawperson proposal at very early stage, aiming to load multiple
+resources efficiently, using a Content-Addressable Bundle, which can link to
+other bundles.
+
+# Authors
+
+- Hayato Ito (hayato@google.com)
+
+# Participate
+
+- [WICG/webpackage](https://github.com/WICG/webpackage/issues)
+
+<!-- TOC -->
+
+## Table of Contents
+
+- [Introduction](#introduction)
+- [Goals](#goals)
+- [Non-Goals](#non-goals)
+- [Web Bundle format](#web-bundle-format)
+- [Web APIs](#web-apis)
+  - [Declarative form](#declarative-form)
+    - [Example](#example)
+  - [How to Load](#how-to-load)
+  - [Navigate to a bundle.](#navigate-to-a-bundle)
+- [Cache strategy](#cache-strategy)
+- [Considered alternatives](#considered-alternatives)
+- [FAQ](#faq)
+- [Stakeholder Feedback / Opposition](#stakeholder-feedback--opposition)
+- [References & acknowledgements](#references--acknowledgements)
+
+<!-- /TOC -->
+
+# Introduction
+
+- Loading many unbundled resources is still slower in 2021. We concluded that
+  [bundling was necessary in 2018](https://v8.dev/features/modules#bundle), and
+  our latest local measurement still suggests that.
+
+- The output of JS bundlers (e.g. webpack) doesn't interact well with the HTTP
+  cache. They are pretty good tools but configuring them to work in an optimal
+  way is tough, and sometimes they're also incompatible with new requirements
+  like
+  [dynamic bundling](https://github.com/azukaru/progressivef-fetching/blob/master/docs/dynamic-bundling/index.md)
```

Comment:
https://github.com/azukaru/progressive-fetching/blob/master/docs/dynamic-bundling/index.md

Reply:
Thanks. Let me fix in another PR!